docs/smtchecker.rst

```diff
@@ -632,7 +632,8 @@ option ``--model-checker-solvers {all,cvc4,eld,smtlib2,z3}`` or the JSON option
 .. note::
   z3 version 4.8.16 broke ABI compatibility with previous versions and cannot
   be used with solc <=0.8.13. If you are using z3 >=4.8.16 please use solc
-  >=0.8.14.
+  >=0.8.14, and conversely, only use older z3 with older solc releases.
```

Comment:
What I meant before was rather that if you're using Solidity >=0.8.14 you should also use the mentioned z3

Reply:
Personally, I was fine with the 'vice versa' version, however, this doesn't make the statement false, right? I.e. a pre `0.8.14` solc will not work with `4.8.16+` z3?

Reply:
Sounds good